README.md

```diff
@@ -27,6 +27,7 @@ Here, I'm collecting a number of links to useful resources for learning JavaScri
 * https://learnjavascript.online/
 * https://adventofcode.com/
 * http://flexboxfroggy.com/ (for CSS, not JavaScript)
+* https://www.codementor.io/javascript-experts
```

Comment:
I don't think this fits in "lessons/challenges"… Those are all meant to be "do-it-yourself" lessons, not "talk to a person". How about adding it to "general" instead? Feel free to add a parenthetical after the URL for some context if appropriate.